src/components/BookmarkCard/Provider.tsx

```diff
@@ -27,11 +28,12 @@ export function Provider({ url, providerUrl, providerName, showUrl }: ProviderPr
 }
 
 function hostname(url: string): string {
-    const { host } = new URL(url);
+    const { host } = new URL(normalizeUrl(url));
     return host;
 }
 
 function homepage(url: string): string {
-    const { origin } = new URL(url);
+    console.log({ url, normalizedUrl: normalizeUrl(url) });
```

Comment:
Do we need log here?

Reply:
Oh, didn't mean to stage this line 🙈 
Thanks for the catch!